include/yaml-cpp/exceptions.h

```diff
@@ -7,6 +7,12 @@
 #pragma once
 #endif
 
+#ifdef _MSC_VER
+#define NOEXCEPT _NOEXCEPT
```

Comment:
I'm uncomfortable defining something so common in a header.

Maybe narrow the scope of the definition and #undef it at the end?